pkg/gen/input/crud/patch.go

```diff
@@ -0,0 +1,17 @@
+package crud
```

Comment:
This file shouldn't be here according to the convention. I guess it isn't clear with the single file examples we have here. I made a PR https://github.com/giantswarm/devctl/pull/59.